ozw-network-visualization-card.js

```diff
@@ -128,24 +129,33 @@ class OZWNetworkVisualizationCard extends HTMLElement {
     }
   }
 
+  //Set different shapes for different device types
   _getShape(device) {
     if (device.node_basic_string === "Static Controller") {
+      return "triangle";
+    } else if (device.node_generic_string === "Binary Switch") {
       return "box";
-    } else if (device.node_basic_string === "Routing Slave") {
+    } else if (device.node_generic_string === "Multilevel Switch") {
       return "ellipse";
-    } else {
+    } else if (device.node_generic_string === "Multilevel Sensor") {
+      return "circle";
+    } else if (device.node_generic_string === "Binary Sensor") {
       return "circle";
+    } else {
+      return "database";
     }
   }
 
   _buildLabel(device) {
     var regDevice = this.device_registry[device.node_id];
-
-    var res = regDevice ? "<b>" + regDevice.name + "</b>\n" : "";
+    //Add user's device name for display
+    var res = regDevice ? "<b>" + regDevice.name_by_user || regDevice.name + "</b>\n" : "";
```

Comment:
this is missing the parenthesis around the logic op
```suggestion
    var res = regDevice ? "<b>" + (regDevice.name_by_user || regDevice.name) + "</b>\n" : "";
```

Reply:
btw, for these suggestions, you can just click on the "Commit suggestion" button to accept :)